docs/development/decision-records/2024-10-25-make-release-manual/README.md

```diff
@@ -0,0 +1,33 @@
+# Make Release Workflow Manual
+
+## Decision
+
+The release workflow will now require a manual activation vs. being automatically triggered via a PR merge event.
+
+## Rationale
+
+Having an automatic trigger has a couple of implications:
+
+- GH only allows for the `on pull_request` event to be of type `closed`. This works fine when a PR is merged but will
+  also trigger
```

Comment:
```suggestion
- GH only allows for the `on pull_request` event to be of type `closed`. This works fine when a PR is merged but it will
  also trigger the event
```